src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,8 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+    const workerName = employees.find(
```

Comment:
I would rename this constant, as it isn't equal to the workers name but is instead equal to to the full worker object, you are just finding the worker by their name.